packages/block-editor/src/components/spacing-sizes-control/index.js

```diff
@@ -29,6 +30,8 @@ export default function SpacingSizesControl( {
 	splitOnAxis = false,
 	useSelect,
 	minimumCustomValue = 0,
+	onMouseOver = noop,
+	onMouseOut = noop,
```

Comment:
Same as previous comments — the default value of `noop` should not be needed here

Reply:
done